pkgs/by-name/wa/wakapi/package.nix

```
@@ -4,22 +4,23 @@
   fetchFromGitHub,
   nixosTests,
   nix-update-script,
+  go_1_25,
 }:
 let
-  version = "2.14.1";
+  version = "2.15.0";
 in
-buildGoModule {
+buildGoModule.override { go = go_1_25; } {
```

This should use `buildGoLatestModule` instead
```suggestion
buildGoLatestModule {
```